src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/BreachResponse.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.probation.courtcaseservice.controller.model;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDate;
+
+@Builder
+@Getter
+public class BreachResponse {
```

Comment:
Need that non-null annotation, can get responses like this 

`{
  "breachId": 2500018597,
  "incidentDate": "2019-09-02",
  "started": "2019-10-02",
  "provider": "NPS North East",
  "team": "Unallocated Team(N02)",
  "officer": "Unallocated Staff(N02) Staff",
  "status": null,
  "order": "CJA - Indeterminate Public Prot."
}`